### ISSUE TITLE
Constant NoSuchField error on startup with Baubles-dependent mods

### ISSUE THREAD

**Issue by genderneutralnoun** [Azanor/Baubles#297]
So, I get into modding Minecraft, I find Botania and its requirement, Baubles. I get a setup with no problems and play for a while. Today I try to load up Forge and it crashes on startup. I fiddle around a bit and the problem seems to be Baubles. I am using the latest version of everything Baubles-related and the game loads perfectly fine without Baubles. I also tried loading the game without the one that seemed to cause it (Botania) but then I just got another error with another mod and Baubles. Here is a crash report with Botania: https://pastebin.com/288J52Fe Here is a crash report without Botania: https://pastebin.com/uVmcmume Botania is the only mod I know of that I have that requires Baubles, but I do have some that integrate with Baubles. Please, I just want to play Botania, and I've been at this for hours because loading Forge takes forever.

**Comment by KingLemming** [comment]
Your jar is out of date. Please update it.

**Comment by genderneutralnoun** [comment]
I am using the latest version from Curseforge for both Baubles and Botania, as in I literally downloaded it and tried again after seeing your comment. Is there a version I don't know about? Another way to update? I also added VanillaFix which makes a guess about which mod could be causing the crash, and it said both Baubles and Draconic Evolution. I will try without Draconic Evolution, but I'm pretty sure I have tried without it already. 
Update: I tried without Draconic Evolution and it gave me an error with Baubles and The Twilight Forest this time. I think it's basically the same thing happening again.
